SECURITY.md

```diff
@@ -2,7 +2,7 @@
 
 ## Supported Versions
 
-The current `master` and `release-pismo` branches are supported with security updates.
+The current `master` and latest `agoric-upgrade-*` branches are supported with security updates.
```

Comment:
I'm not sure what the correct descriptor should be, should we include latest and pre-release tagged releases in https://github.com/Agoric/agoric-sdk/releases ?

Reply:
Branch of that shape is definitely not correct, it should be `agoric-upgrade-*` **tags**, but as you mention without restrictions it would include outdated releases that may have known issues. Maybe something like 
> The current `master` branch and the `agoric-upgrade-*` tags corresponding to the latest [release and pre-release versions](https://github.com/Agoric/agoric-sdk/releases).

PS: We do have `release-*` branches, but those are mostly to stage the creation of these release tags, and may similarly become outdated like `release-pismo` now is.